.github/workflows/publish_docs.yml

```diff
@@ -23,4 +23,4 @@ jobs:
       uses: sphinx-notes/pages@v3
       with:
         documentation_path: ./docs/source/
-        requirements_path: ./docs/source/requirements.txt
+        pyproject_extras: docs
```

Comment:
Does this not require all extras to make all the modules importable?

Reply:
The `docs` dependency group includes all extras, so it should work as expected, but the deployment workflow is opaque and only triggered on commits to `master` so I can't be sure.